aubio-rs/src/vec.rs

```diff
@@ -13,6 +12,9 @@ use std::{
     ptr::null_mut,
 };
 
+use std::ptr;
+use std::slice;
```

Comment:
Pls, merge this imports.

Reply:
You should merge this with above import section.

Reply:
Ha just understood, sorry. It's gone now